Cargo.toml

```diff
@@ -38,3 +38,7 @@ geo-types = { version = "0.7.10", features = ["approx"] }
 [package.metadata.docs.rs]
 features = [ "proj-sys/nobuild", "network", "geo-types" ]
 rustdoc-args = ["--cfg", "docsrs"]
+
+
+[patch.crates-io]
+libsqlite3-sys = { git = "https://github.com/rusqlite/rusqlite/", rev = "42a82fb" }
```

Comment:
This currently relays on an accepted but unreleased patch to the libsqlite3-sys crate: https://github.com/rusqlite/rusqlite/pull/1441

Reply:
This is now published as libsqlite-sys 0.28, therefore I just updated the version and removed that patch